CONTRIBUTING.md

```diff
@@ -120,3 +120,82 @@ TiDB 中文文档使用 Markdown 语言进行编写，为了保证文档质量
 26. [MD046 - Code block style](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md046---code-block-style)
 
     文档中代码块统一使用**三个反引号**进行包裹，**禁止**使用**缩进四格**风格的代码块。(`style`=`fenced`)
+
+## Pull Request 提交流程
+
+TiDB 文档的修改需要遵循一定的流程，具体如下。
+
+> **注意：**
+>
+> 目前 TiDB 主要维护三个版本的文档：dev（最新开发版），v3.0（最新稳定版），v2.1（最新 2.1 版）。提 Pull Request 前请务必考虑修改会影响的文档版本，并据此修改所有相应的版本。
+
+### 第 1 步：Fork pingcap/docs-cn 仓库
+
+1. 打开 pingcap/docs-cn 项目[仓库](https://help.github.com/articles/github-glossary/#repository)：<https://github.com/pingcap/docs-cn>
+2. 点击右上角的 [**Fork**](https://help.github.com/articles/github-glossary/#fork) 按钮，等待 Fork 完成即可。
+
+### 第 2 步：将 Fork 的仓库克隆至本地
+
+```
```

Comment:
```suggestion
```shell
```
要不要加个bash或shell高亮一下代码？

Reply:
@yikeke This doc won't be displayed on the website.